### DISCUSSION TITLE
Having trouble setting up Signal Strategy

### DISCUSSION THREAD

**Discussion by jy13249** [kernc/backtesting.py#492] [Q&A]
So i set up a strategy that i want to use for creating entry/exit signals.
I'm trying to see all the signals that get generated.
I'm not sure if im not setting it up right or if im not looking at the right thing, but the Signals series that comes back just brings back all 0's. I tried looking in the docs for help setting it up but i cant get the hang of it. Any help would be appreciated!

![image](https://user-images.githubusercontent.com/62567532/135687208-941b3bdf-8e5a-43c6-9717-8180f2b0abb4.png)

**Comment by kernc** [comment]
Not sure about all zeros (it's literally the values of your conjunction condition), but [`SignalStrategy.set_signal()`](https://kernc.github.io/backtesting.py/doc/backtesting/lib.html#backtesting.lib.SignalStrategy.set_signal) was supposed to be called in `init()` only.
